src/components/App.tsx

```diff
@@ -40,48 +34,41 @@ const App: FC = () => {
         window.innerHeight - headerElement.current.clientHeight + 'px';
   }, [selectPathway]);
 
-  function setEvaluatedPathwayCallback(
-    value: EvaluatedPathway | null,
-    selectPathway = false
-  ): void {
+  function setPathwayCallback(value: Pathway | null, selectPathway = false): void {
     window.scrollTo(0, 0);
     setSelectPathway(selectPathway);
     setCurrentPathway(value);
   }
 
-  const updateEvaluatedPathways = useCallback(
-    (value: EvaluatedPathway) => {
-      const newList = [...evaluatedPathways]; // Create a shallow copy of list
-      for (let i = 0; i < evaluatedPathways.length; i++) {
-        if (evaluatedPathways[i].pathway.name === value.pathway.name) {
+  const updatePathways = useCallback(
+    (value: Pathway) => {
+      const newList = [...pathways]; // Create a shallow copy of list
+      for (let i = 0; i < pathways.length; i++) {
```

Comment:
I know the list here isn't long but we should consider making the pathways state a map instead of a list

Reply:
Upon closer review we can even delete this whole update function -- originally it was used when the ELM evaluation returned but now we don't need that anymore. I can still refactor the pathways state to be a map if you really want, but there's no longer any access to the list by name anymore.